.travis.yml

```diff
@@ -3,6 +3,7 @@ php:
   - '7.1'
   - '7.2'
   - '7.3'
+  - '7.4'
   - 'nightly'
```

Comment:
I would drop this to make tests pass. There is actually no need to test against PHP 8.0 imho

Reply:
Good point. I moved nightly to `allow failures`